### PR TITLE
Mapfrontend2

### PR DESCRIPTION
**Pull request by ghost**
fixed marker load bug, added logo and site title, started to incorporate news functionality

### CODE REVIEW

File: appserver/website-bootstrap/index.html [hunk 1]
```diff
@@ -94,13 +99,13 @@ <h3 class="side-title">Top News</h3>
...
-                <li><a href="">Scott Salisbury</a></li>
-                <li><a href="">Dan Tran</a></li>
-                <li><a href="">Thanh Tran</a></li>
+                <li>Jennifer Chao</li>
+                <li>Conor Kennedy</li>
+                <li>Mike Lauw</li>
+                <li>Chris Nelson</li>
+                <li>Scott Salisbury</li>
+                <li>Dan Tran</li>
+                <li>Thanh Tran</li>
```
Comment (BareBeaverBat) [inline appserver/website-bootstrap/index.html:108]: why no links?

File: appserver/website-bootstrap/main.css [hunk 2]
```diff
@@ -382,105 +393,4 @@ body {
...
-	background-color: #fff;
-	padding-top: 8px;
-	padding-bottom: 8px;
-	padding-left: 20px;
-	padding-right: 20px;
-	font-size: 18px;
-	font-weight: 600;
-	font-color: #2e2e4d;
-	border-radius: 4px;
-	cursor: pointer;
```
Comment (BareBeaverBat) [inline appserver/website-bootstrap/main.css]: I think you need to merge into your branch the latest commits in master (from Mike's temporal UI input elements branch)

**Review by BareBeaverBat — CHANGES_REQUESTED** [review]

**Comment by BareBeaverBat** [comment]
Awesome work- thanks for clearing up those bugs so quickly!

File: appserver/website-bootstrap/index.html (hunk 1 above)
Comment (ghost) [reply appserver/website-bootstrap/index.html:108]: no one seemed interested, we can add them easily later if people are interested

File: appserver/website-bootstrap/main.css (hunk 2 above)
Comment (Cnelson10) [reply appserver/website-bootstrap/main.css]: should be good to go now, should I make another pull request?

Comment (ghost) [reply appserver/website-bootstrap/main.css]: added another commit, containing this code

**Review by BareBeaverBat — APPROVED** [review]

**Pull request merged**